Assets/Scripts/OrchestraArmy/Entity/Entities/Players/Player.cs

```diff
@@ -1,31 +1,35 @@
 using OrchestraArmy.Entity.Controllers;
 using OrchestraArmy.Entity.Entities.Players.Controllers;
+using OrchestraArmy.Entity.Entities.Players.WeaponSelection;
+using OrchestraArmy.Event;
+using OrchestraArmy.Event.Events.Player;
+using UnityEngine;
 
 namespace OrchestraArmy.Entity.Entities.Players
 {
-    public class Player : LivingDirectionalEntity
+    public class Player : LivingDirectionalEntity, IListener<PlayerDamageEvent>
     {
         /// <summary>
         /// The controller for the player's camera.
         /// </summary>
         public ICameraController CameraController { get; set; }
 
-        /// <summary>
-        /// The controller for the player's movement.
-        /// </summary>
-        public IMovementController MovementController { get; set; }
-
         /// <summary>
         /// The controller for the player's attacking.
         /// </summary>
         public IAttackController AttackController { get; set; }
         
+        /// <summary>
+        /// The player's weapon selection wheel.
+        /// </summary>
+        public WeaponWheel WeaponWheel { get; set; }
         
         protected override void Update()
         {
             base.Update();
             DirectionController.HandleDirection();
             SpriteManager.UpdateSprite();
+            AttackController.HandleAttack();
```

Comment:
Is het handig om dit hier te doen? Idk, maar lijkt me dat attacks ook physics hebben & dat zet je meestal in FixedUpdate

Reply:
Spawnt op dit moment een nieuw object dat z'n eigen cycle heeft, dus doet niet echt wat met physics

Reply:
Dan staat 'ie goed.